lib/next_rails/bundle_report/cli.rb

```diff
@@ -3,6 +3,7 @@
 require "optparse"
 require "next_rails"
 require "next_rails/bundle_report"
+require 'byebug'
```

Comment:
❤️  love it. since all above has double-quotes, let's go with that please
```suggestion
require "byebug"
```